src/components/desktop/desktop-toolbar.vue

```diff
@@ -49,5 +47,12 @@
       ColumnPicker
     }
   })
-  export default class DesktopToolbar extends Vue {}
+  export default class DesktopToolbar extends Vue {
+    protected get query(): string {
+      if (this.$x.spellcheckedQuery) {
+        return this.$x.spellcheckedQuery;
+      }
+      return this.$x.query.searchBox;
```

Comment:
Could you change it to a ternary?

```suggestion
      return this.$x.spellcheckedQuery ? this.$x.spellcheckedQuery : this.$x.query.searchBox;
```